### ISSUE TITLE
ERROR: You must not use 8-bit bytestrings

### ISSUE THREAD

**Issue by 0x3F3F** [pictuga/morss#23]
Hi,

I've tried to install morss but have hit upon an issue when running the test scenario:

```
MyMachine$ python -m debug http://www/bbc/co/uk/bla/bla.xml
ERROR: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.
```

[Stackoverflow ](https://stackoverflow.com/questions/3425320/sqlite3-programmingerror-you-must-not-use-8-bit-bytestrings-unless-you-use-a-te) suggested adding ".text_factory = str" in, which I did in the init function of the SQLiteCache function in crawlers.py, which appears to have solved my problem. Later comments  suggest this isn't the right solution and the data should be converted to binary:

I'm not sure what's right, but thought I'd flag it up to you. I've not had a chance to use this properly yet, but looks very useful - thanks for sharing it.

Cheers,
Iain.

**Comment by 0x3F3F** [comment]
When running I'm also getting frequent errors when I try to run it.  I think related to the above:

ERROR: initial_value must be unicode or None, not str

**Comment by pictuga** [comment]
Indeed, solved with f9217102f37b0 (forgot to push the commit to github, my bad)
